com.sejda.Desktop.yml

```diff
@@ -0,0 +1,58 @@
+id: com.sejda.Desktop
```

Comment:
`Desktop` is a bad application name. With or without uppercase `D`.

Reply:
and you don't need to close the PR to change the ID. the branch name is most irrelevant.

Reply:
App id was changed to`com.sejda.Sejda`